client/src/components/charts/DashMap.jsx

```diff
@@ -6,14 +6,19 @@ require('../../../node_modules/leaflet/dist/leaflet.css');
 require('../../styles/DashMap.scss');
 
 
-export default function DashMap({ visualisation, datasets }) {
+export default function DashMap({ visualisation, datasets, width, height }) {
   const chartData = chart.getChartData(visualisation, datasets);
   return (
     <div className="DashMap dashChart">
       <Map
         center={[0, 0]}
         zoom={2}
         scrollWheelZoom={false}
+        key={width}
```

Comment:
I *think* that adding this key triggers a deeper re-render when the key changes - which is desirable with the maps visualisation because often, when the width changes, new tiles have to be loaded. Adding the key appeared to improve dynamic tile-loading on resizing, but I will test this further.